### ISSUE TITLE
WFS data not shown on map

### ISSUE THREAD

**Issue by magerlin** [tomchadwin/qgis2web#530]
I have a Qgis project with combinations of WMS and WFS layers, but WFS layers (The red, yellow and green dots) are not shown on the produced map: [WFS](http://byogtrafik.ramboll.dk/SandBox/Qgis2Web/WFS/)

If I instruct Qgis2web to "Encode to JSON" the map works as desired (except for being a "dead" map not updated by WFS): [Json](http://byogtrafik.ramboll.dk/SandBox/Qgis2Web/Json/)

When not encoding to Json Qgis2web shows this console error:
Console error:
filter=Failed to load resource: Error opening C:/Users/MAP/AppData/Local/Temp/qgis2web/qgis2web_2017_08_22-13_17_56_557000/filter=: No such file or directory

Project file is here: 
[Mastra.zip](https://github.com/tomchadwin/qgis2web/files/1241880/Mastra.zip)

I an using lastest Qgis2web version 2.22.

I could produce the map without the above problems with the version that was the current version on April 4th 2016.

**Comment by tomchadwin** [comment]
It's the OGC filter you are using in your WFS layers - I'm afraid filters are not supported: https://github.com/tomchadwin/qgis2web/issues/102.

**Comment by magerlin** [comment]
OK - I think it has been working earlier as I wrote. Unfortunately I do not have the working map anymore...

**Comment by tomchadwin** [comment]
If you can remove the filter in QGIS, see whether that works.

**Comment by magerlin** [comment]
Removing the filter makes no change - It is still not being displayed.

The console error from a complete blank Qgis project with just the unfiltered WFS shows this error:

SyntaxError: Parse error
ows:-1Resource interpreted as Script but transferred with MIME type application/xml.

**Comment by tomchadwin** [comment]
Can you upload that last project?

**Comment by magerlin** [comment]
Sure - it is here: 
[WFS.zip](https://github.com/tomchadwin/qgis2web/files/1242231/WFS.zip)

**Comment by tomchadwin** [comment]
It works for me:

![2017-08-22 15_04_15-index html](https://user-images.githubusercontent.com/7840213/29569414-59057f98-874b-11e7-9113-0489b2514c68.png)

**Comment by tomchadwin** [comment]
Is it possible that you suffered connection issues? It's a big WFS layer, so it takes a while to download. I'll close this for now, since it worked for me. I'll reopen if you can get any other info to us.

**Comment by magerlin** [comment]
It is working now for me too (without the filtering) - do not now what happened yesterday?

**Comment by tomchadwin** [comment]
Very strange. Pleased it's working. Sorry about the filter issue - I can't think of a solution to it yet.

**Comment by magerlin** [comment]
I started up from a new empty project - I often experience that some traces are left in project files even if I delete layers from the project. It might have been the issue in this situation.  

I have tried using a rule based styling as filtering and it was somehow working except for that I want to use clustering in the Leaflet map. Problem is that the initial clusters will also include the points I have filtered away in the styling so some clusters will "explode into nothing" when I zoom in (as the points are filtered away in the styling).
